src/mame/drivers/hp80.cpp

```diff
@@ -0,0 +1,786 @@
+// license:BSD-3-Clause
+// copyright-holders:F. Ulivi
+//
+// *******************************
+// Driver for HP series 80 systems
+// *******************************
+//
+// This is WIP: lot of things still missing
+
+#include "emu.h"
+#include "screen.h"
+#include "cpu/capricorn/capricorn.h"
+
+// Debugging
+#define VERBOSE 1
+#include "logmacro.h"
+
+// Bit manipulation
+static constexpr unsigned BIT_MASK(unsigned n)
+{
+	return 1U << n;
+}
+
+template<typename T> void BIT_CLR(T& w , unsigned n)
```

Comment:
Same for these templates - place them in an anonymous namespace to stop the being exported.